.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           jina check
           export JINA_LOG_VERBOSITY="ERROR"
           pip install .[test]
-          pytest --force-flaky --min-passes 1 --cov=jina --cov-report=xml -n 1 --timeout=120 -v tests/unit
+          pytest --force-flaky --min-passes 1 --cov=jina --cov-report=xml -n 1 --timeout=600 -v tests/unit
```

Comment:
Is there any reason why we need to increase the timeout for each test?

Reply:
I guess is because we are suffering quite often from timeouts

Reply:
Earlier CI was failing due to timeout of test

Reply:
please revert this change, the timeout is a false alarm. it is the port conflict problem caused by flaky tests.